### ISSUE TITLE
Glu-C specificity

### ISSUE THREAD

**Issue by fcsigloch** [MSGFPlus/msgfplus#44]
The specificity of glutamyl endopeptidase (Glu-C) is dependent on the buffer used for digestion. In ammonium bicarbonate and ammonium acetate, the enzyme specificity is higher at the glutamic residues (E). In phosphate buffers, cleavage occurs at the aspartic (D) and glutamic residues (E).

When I analysed samples digested with Glu-C in phosphate buffer, MSGFPlus seems to find only peptides ending with glutamic acid (E). How is the specificity of Glu-C defined in MSGFPlus? If cleavages behing D are not allowed, could you provide the specificity of Glu-C in phosphate buffers as an additional "enzyme"?

**Comment by FarmGeek4Life** [comment]
https://github.com/MSGFPlus/msgfplus/blob/master/src/main/java/edu/ucsd/msjava/msutil/Enzyme.java#L386 shows the definition of Glu-C that MSGPlus uses. You can use some of the extensibility built into MSGFPlus to define another enzyme that matches what you are trying to do; instructions for that are at https://htmlpreview.github.io/?https://raw.githubusercontent.com/MSGFPlus/msgfplus/master/doc/ScoringParamGen.html, under the heading `How to make a parameter file for a new fragmentation method or a new enzyme?`

**Comment by lanyuchunmo** [comment]
yes, I also neet to set a customerized enzyme, I learned the method of customerized enzyme cleavages, for example, I add a line in the enzymes.txt file: 
A1101C,KF,C,A1101-C
and then copy this file to the params directory to current analysis directory and then ran the MSGFPlus with no parameter and check the id of the newly seted cleavage (the id is 10), lastly, I ran the MSGFPlus with the following commands:
java  -Xmx10000M   -jar MSGFPlus.jar   -s data.mgf   -d uniprot_all_human_20171220_contaminant.fasta   -o target_search.mzid   -t 10ppm   -ti -1,2   -thread 16   -tda 0   -ntt 0   -m 3   -inst 3   -e 10   -minLength 8   -maxLength 15   -addFeatures 1
java -Xmx20000M   -cp MSGFPlus.jar   edu.ucsd.msjava.ui.MzIDToTsv   -i target_search.mzid   -o target_search.tsv   -showQValue 1   -unroll 0

BUT, BUT, I found that the newly set cleavage does not seem to work, many of the peptides are not ending with K/F !!!
then I test with "-e 1" parameters, the resulting peptides are mostly ending with K/R (the trypsin cleavage site).
Why it come out like this?
